src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -311,6 +327,33 @@ export const useBalanceChartData: UseBalanceChartData = args => {
   const balances = useAppSelector(state =>
     selectPortfolioCryptoBalancesByAccountIdAboveThreshold(state, accountId)
   )
+
+  // Get total delegation
+  // TODO(ryankk): consolidate accountSpecifiers creation to be the same everywhere
+  const cosmosCaip2: CAIP2 = caip2.toCAIP2({
+    chain: ChainTypes.Cosmos,
+    network: NetworkTypes.COSMOSHUB_MAINNET
+  })
+
+  const accountSpecifiers = useSelector(selectAccountSpecifiers)
+  const account = accountSpecifiers.reduce((acc, accountSpecifier) => {
+    if (accountSpecifier[cosmosCaip2]) {
+      acc = accountSpecifier[cosmosCaip2]
+    }
+    return acc
+  }, '')
+
+  // TODO(ryankk): this needs to be removed once staking data is keyed by accountSpecifier instead of caip10
+  const cosmosCaip10 = account ? caip10.toCAIP10({ caip2: cosmosCaip2, account }) : ''
+
+  // load staking data to redux state
+  useGetStakingDataQuery({ accountSpecifier: cosmosCaip10 })
+
+  const delegationTotal =
+    useAppSelector(state =>
+      selectTotalStakingDelegationCryptoByAccountSpecifier(state, cosmosCaip10)
+    ) ?? '0'
```

Comment:
shouldn't have to optchain on this selector, it should do that logic and always return a string